docs/modules/ROOT/pages/usage.adoc

```diff
@@ -70,6 +70,11 @@ spec:
     name: simple-zk
 ----
 
+== Monitoring
+
+The managed NiFi instances are automatically configured to export Prometheus metrics. See
+xref:home::monitoring.adoc[] for more details.
```

Comment:
The "Monitoring" page doesn't contain more details, either we add some details there (Which port? What metrics are exported?) or we remove that page (It only contains the sentence "As of now enabling monitoring is currently disabled.")

Reply:
The monitoring page is actually obsolete, this points to https://docs.stackable.tech/home/monitoring.html

Reply:
Oh alright!